06-management/03-migration-and-backup.md

```diff
@@ -0,0 +1,110 @@
+---
+pageTitle: Migration and Backup
+keywords: grakn, migration, backup
+longTailKeywords: grakn migration
+Summary: Backing up and migrating data between versions of Grakn.
+toc: false
+---
+
+# Migration and Backup
+
+Some versions of Grakn include tools to backup and migrate data between versions of grakn that are not data-level compatible. Please note the chart below for which versions are data-level compatible:
+
+| Data-level Versions | Versions at Data-level | Latest Version With Migration Available |
+| ------------------- | ---------------------- | --------------------------------------- |
+| 1.7                 | 1.5.x, 1.6.x, 1.7.x    | 1.7.3                                   |
```

Comment:
might be worth confirming that 1.5.x works by downloading a distro and trying to copy the DB after inserting some data!